### PR TITLE
[TECHNICAL-SUPPORT] LPS-85076

### PR DESCRIPTION
**Pull request by sergiogonzalez**
/cc @fortunatomaldonado

### CODE REVIEW

File: modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs/js/blogs.js [hunk 1]
```diff
@@ -441,7 +441,9 @@ AUI.add(
 													var cancelButton = instance.one('#cancelButton');
```
Comment (sergiogonzalez) [inline modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs/js/blogs.js]: @ambrinchaudhary I think that we don't need any of this logic that updates the cancelButton link. I think that we could safely remove all this and we could also remove this line as well https://github.com/liferay/liferay-portal/blob/master/modules/apps/blogs/blogs-web/src/main/java/com/liferay/blogs/web/internal/portlet/action/EditEntryMVCActionCommand.java#L227

It seems that the url that does the autosaving (which is set here https://github.com/liferay/liferay-portal/blob/master/modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs/edit_entry.jsp#L485) doesn't have the "redirect" parameter, so when the request is sent to the server, the server won't include the "redirect" parameter in the json object and the javascript will change the cancel link to a empty value. However, I don't see why the server needs to send the redirect parameter in the json so the javascript can change it. I think that the cancel button should have the link when it's rendered and we shouldn't change that at any time.

Can you confirm this? is there anything that I'm missing?

**Comment by ambrinchaudhary** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs/js/blogs.js (hunk 1 above)
Comment (ambrinchaudhary) [reply modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs/js/blogs.js]: I completely agree with you @sergiogonzalez. Cancel button link shouldn't be changed in the autosave, it seems this logic is quite obsolete and there is no need of that now.

**Comment by ambrinchaudhary** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/62904. [See changes here.](https://github.com/ambrinchaudhary/liferay-portal/compare/3195637a~1...3195637a) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**